client/components/common/header/Header.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component, PropTypes } from 'react';
+import { IndexLink } from 'react-router';
+import { connect } from 'react-redux';
+import toastr from 'toastr';
+import MainMenu from './MainMenu.jsx';
+import { logout } from '../../../actions/userActions';
+
+class Header extends Component {
+  constructor(props) {
+    super(props);
+
+    this.logout = this.logout.bind(this);
+  }
+
+  static componentWillMount() {
```

Comment:
Thanks for the review of the application. I used `static` to disable the eslint error requesting that `this` be used in the function